app/models/service_template.rb

```diff
@@ -5,6 +5,7 @@ class ServiceTemplate < ActiveRecord::Base
   include NewWithTypeStiMixin
   include TenancyMixin
   include_concern 'Filter'
+  CLOUD_PROVIDERS = %w(amazon openstack)
```

Comment:
This is going to be a pain to maintain, particularly with respect to pluggability.  There should be a way to determine based on the models using something like `model_for(prov_type).kind_of?(ManageIQ::Providers::CloudManager)` given you have the model or can create a method like `model_for`